lib/doorkeeper/config.rb

```diff
@@ -14,7 +14,7 @@ def self.configure(&block)
   end
 
   def self.configuration
-    @config || (fail MissingConfiguration.new)
+    @config || (fail MissingConfiguration)
```

Comment:
No, because message is initialized in the constructor (see [config.rb#L3](https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/config.rb#L3)). You can check it in _irb_/_rails console_:

```ruby
2.3.0 :004 > raise Doorkeeper::MissingConfiguration
# Doorkeeper::MissingConfiguration: Configuration for doorkeeper missing. Do you have doorkeeper initializer?
```